lldb/source/Host/common/TCPSocket.cpp

```diff
@@ -115,6 +115,18 @@ std::string TCPSocket::GetRemoteConnectionURI() const {
   return "";
 }
 
+std::vector<std::string> TCPSocket::GetListeningConnectionURI() const {
+  if (m_listen_sockets.empty())
+    return {};
+
```

Comment:
```suggestion
```

The common code handles this case just fine.

Reply:
Applied suggestion